### PR TITLE
Set Metering unit TEST_OUTPUT_PATH to allow storing junit test report results and coverage results in artifacts dir

### PR DESCRIPTION
**Pull request by chancez**
Adding support to produce junit report results, this configures the tests to output them to the correct location.

### CODE REVIEW

File: ci-operator/config/operator-framework/operator-metering/operator-framework-operator-metering-master.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ build_root:
     context_dir: .
 tests:
 - as: unit
-  commands: make unit
+  commands: make unit TEST_OUTPUT_PATH=/tmp/artifacts
```
Comment (tschuy) [inline ci-operator/config/operator-framework/operator-metering/operator-framework-operator-metering-master.yaml]: shouldn't this be TEST_OUTPUT_DIR? 

https://github.com/operator-framework/operator-metering/pull/605/files#diff-3013a7c42949980d2af09a722fb3f5beR7

Comment (chancez) [reply ci-operator/config/operator-framework/operator-metering/operator-framework-operator-metering-master.yaml]: Oops yes

**Pull request merged**